files/en-us/glossary/client_hints/index.html

```diff
@@ -35,4 +41,6 @@ <h2 id="See_also">See also</h2>
 <ul>
  <li><a href="/en-US/docs/Web/HTTP/Headers#client_hints">Client Hints headers</a></li>
  <li><a href="/en-US/docs/Web/HTTP/Headers/Vary"><code>Vary</code> HTTP Header</a></li>
+ <li><a href="https://wicg.github.io/client-hints-infrastructure/">Client Hints Infrastructure, W3C Draft Community Group Report, 29 April 2021</a></li>
```

Comment:
This now has May 17 as a date for me and probably changes frequently. I don't think it is useful to add this.
```suggestion
 <li><a href="https://wicg.github.io/client-hints-infrastructure/">Client Hints Infrastructure</a></li>
```